lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -198,15 +198,27 @@ combineDbAndLsqData
     -> Map PoolId PoolDbData
     -> Map PoolId Api.ApiStakePool
 combineDbAndLsqData sp =
-    Map.merge lsqButNoChain chainButNoLsq bothPresent
+    Map.merge lsqButNoDb dbButNoLsq bothPresent
   where
-    lsqButNoChain = traverseMissing $ \k lsq -> pure $ mkApiPool k lsq Nothing
-
-    -- In case our chain following has missed a retirement certificate, we
-    -- treat the lsq data as the source of truth, and dropMissing here.
-    chainButNoLsq = dropMissing
+    lsqButNoDb = traverseMissing $ \k lsq ->
+        pure $ mkApiPool k lsq Nothing
+
+    -- When a pool is registered (with a registration certificate) but not
+    -- currently active (and therefore not causing pool metrics data to be
+    -- available over local state query), we use a default value of /zero/
+    -- for all stake pool metric values so that the pool can still be
+    -- included in the list of all known stake pools:
+    --
+    dbButNoLsq = traverseMissing $ \k db ->
+        pure $ mkApiPool k lsqDefault $ Just db
+      where
+        lsqDefault = PoolLsqData
+            { nonMyopicMemberRewards = minBound
```

Comment:
This needs to be checked / justified against the specs. I am not sure that `0` would be a significant value for new pools.

Reply:
@jonathanknowles after checking, the ledger team suggests that (given the time constraints) we use the average non myopic member rewards of the top `k` pools (where `k` is the desired number of pools).